source/administration/devops-command-center.rst

```diff
@@ -181,12 +187,12 @@ Incident status updates ensure that stakeholders remain informed about the progr
 1. Select the **Incidents** icon in the channel header to open the right-hand sidebar.
 2. Select the **Summary** tab.
 3. Within the **Summary** tab, select **Update Status**.
-4. Add a markdown-formatted message.
+4. Add a Markdown-formatted message.
  * If this is the first status update and the playbook has a defined template, that template will be pre-populated here.
  * If this is a subsequent status update, the message from the last status update will be pre-populated here.
 5. Optionally set a reminder to prompt for the next status update.
- * If this is the first status update and the playbook has a defined default reminder timer, that timer will be pre-selected here.\
- * If this is a subsequent status update, the last reminder timer will be pre-populated here.\
+ * If this is the first status update and the playbook has a defined default reminder timer, that timer will be pre-selected here.
+ * If this is a subsequent status update, the last reminder timer will be pre-populated here.
 6. Select **Update Status** to post your status update.
  * Status updates are posted to the incident channel as a message from the user providing the status update.
  * If the playbook has a defined broadcast channel, status updates are copied to the broadcast channel as a message from the incident bot.
```

Comment:
Below here:
- Ending an incident needs updating. Reference to stage and it's simpler now. No need to be on last stage, etc. 
- Same with restart. 
- Stages still referenced in the slash commands section. 

Unrelated:
- Why are we documenting the test data command on user facing documentation?

Reply:
Good question re: the test data section. When this was moved across from the Gitbook docs to here I left it in - however, it's very easy to move it back to the repo/Gitbook and have it in the development section. Thoughts?

Reply:
Moving it back sounds good.

Reply:
It's moved. :)